mathml/elements/menclose.json

```diff
@@ -145,9 +145,27 @@
               },
               "chrome_android": "mirror",
               "edge": "mirror",
-              "firefox": {
-                "version_added": "1"
-              },
+              "firefox": [
+                {
+                  "version_added": "32",
+                  "notes": "Notation <code>phasorangle</code> was implemented."
+                },
+                {
+                  "version_added": "24",
+                  "partial_implementation": true,
+                  "notes": "Notation <code>updiagonalarrow</code> was implemented."
+                },
+                {
+                  "version_added": "2",
+                  "partial_implementation": true,
+                  "notes": "Notation <code>madruwb</code> was implemented."
+                },
+                {
+                  "version_added": "1",
+                  "partial_implementation": true,
+                  "notes": "Only notations from MathML2 were implemented."
+                }
+              ],
```

Comment:
I think it would be better to place these into subfeatures, so that the compatibility statements don't get so long here?

Reply:
OK, but I guess that means adding keys for each browser again? Also menclose is not in MathML Core and its future in browsers is uncertain, so not sure it's worth spending too much time on it... Moreover, these notes are for very old versions of Firefox. Maybe we can just keep version 1 (partial) and 32 (complete) instead?

Reply:
Added subfeatures.